### ISSUE TITLE
System error 67 has occurred

### ISSUE THREAD

**Issue by gcraenen** [rapid7/metasploitable3#269]
## Issue Description
System error 67 has occurred

## Host System
* OS: MacOS X High Sierra
* Packer Version:
* Vagrant Version:
* VirtualBox Version: 5.2.8 r121009

## Command Output
win2k8: Running provisioner: shell...
    win2k8: Running: scripts/installs/setup_linux_share.bat as c:\tmp\vagrant-shell.bat
    win2k8: CMDKEY: Credential added successfully.
    win2k8: System error 67 has occurred.
    win2k8: The network name cannot be found.
The following WinRM command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

cmd /q /c "c:\tmp\vagrant-shell.bat"

Stdout from the command:

CMDKEY: Credential added successfully.

Stderr from the command:

System error 67 has occurred.

The network name cannot be found.

**Comment by duncan426** [comment]
I just got the same error as well on my Windows box.

**Comment by Zero-Nem** [comment]
I sorted this issue by logging into the VBox VM using the vagrant account and creating a folder called 'public' in the root directory of the C: drive. Next I shared the folder (right-click the folder -> properties -> sharing -> share -> share). To test the share, I opened a command prompt and ran the following: net use W: \\172.28.128.3\public - the command ran successfully. 

Outside the VM , in my powershell terminal I re-ran 'vagrant provision win2k8' and the provisioning process completed successfully. 

===============
FYI-: for troubleshooting purposes, if you run the VM and navigate to C:\tmp\ at the time you're receiving the error, you will see the 'vagrant-shell.bat' file. Simply open this batch file with notepad and take a look at the contents. The explicit line in the script that was receiving the error response is 'net use W: \\172.28.128.3\public /savecred /p:yes'

If you're wondering where the 172.28.128.3 address is coming from it's being assigned to your VM via DHCP from a virtualbox host network through a vbox 'host-only' adapter. The VM will also have a 10.0.2.X NAT network IP assigned through vbox. 

Running an 'ipconfig' in cmd on the VM will confirm this.

**Comment by oldRedFox** [comment]
I am getting the same error and I did as @Zero-Nem suggested. However, I am not seeing the tmp directory getting created in the VM even though the output states that it is copying files to C:\tmp and appears to be running the batch files. 

I am able to import the box.ovf into VirtualBox and boot into the OS. I did find that the guest OS was assigned an IP from the 10.0.2.X. range. I switched the host-only adapter to the one that did not have DHCP enabled and was configured for 172.28.128.1. That did not work either. I even attempted to manually assign the IP that is used in the setup_linux_share.bat file in the guest.
